src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -1525,8 +1525,15 @@ def update_stack_component(
                         "Service connector with ID "
                         f"{component_update.connector} not found."
                     )
+            else:
+                existing_component.connector = None
+                existing_component.connector_resource_id = None
 
             if service_connector:
+                logger.debug(
+                    "Updating component %s with connector %s",
+                    existing_component,
+                )
```

Comment:
String has two `%s` but you only give one formatting input (connector is not passed)